zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -527,6 +535,145 @@ benchmarks! {
             Ok(())
         })?;
     }: { Pallet::<T>::start_subsidy(&market_clone.unwrap(), market_id)? }
+
+    market_status_manager {
+        let b in 1..31;
+        let f in 1..31;
+        // let t in 0..10;
+
+        // ensure markets exist
+        for _ in 0..64 {
+            let _ = create_market_common::<T>(
+                MarketCreation::Permissionless,
+                MarketType::Categorical(T::MaxCategories::get()),
+                ScoringRule::CPMM
+            )?;
+        }
+
+        let block_number: T::BlockNumber = Zero::zero();
+        let last_time_frame: TimeFrame = Zero::zero();
```

Comment:
not sure why last_time_frame is always zero?

Reply:
It's just an arbitrary [start value](https://github.com/zeitgeistpm/zeitgeist/blob/30b7191666809c5f40293b0e815c9336d3a9ec91/zrml/prediction-markets/src/lib.rs#L2200). More important is the `current_time_frame`, because my initial thought was to iterate only once over the parent for loop. So the `current_time_frame` should be `last_time_frame + 1 == current_time_frame` (assuming the for-loop is inclusive).